### ISSUE TITLE
need to create a menu for the bot that allows you to labeling data

### ISSUE THREAD

**Issue by Ouozen** [ruscaang/type-content-bot#5]
The user path looks like this: if the user sees data incorrectly marked up by the bot, then he makes reply messages with a special command to correct the error. The bot returns a menu where you can select the correct type of content. After that, the bot puts a reaction to the user's message and deletes its message, which contains a menu.

**Comment by Sekai-no-uragawa** [comment]
Need to close this issue first https://github.com/ruscaang/type-content-bot/issues/2
+ To implement this, I think we need a DB with the latest marked links for each user. Seems like a minimum set:

- user_id
- msg_id (id of message with link in chat)
- link_msg_id (id of a message in the archive, with an incorrectly marked link)
- tag_name
- timestamp 

I propose to put it in a separate issue.

**Comment by Ouozen** [comment]
Yes, I also think that it should be implemented after adding the database. I just think it's necessary to implement this function a little wider. It is necessary that messages that the bot has not assigned to any class can also be labeled. It seems that for this you need to have information about the message that was replied in order to perform some actions in the database with it to change the label.

**Comment by Dimildizio** [comment]
Currently the bot labels every message that doesn't fall into any of categories as 'trash', however images, videos, polls, etc are not logged into db
